spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-ecommerce-tracker.spec.js

```diff
@@ -118,6 +118,22 @@ describe('Google Analytics 4 ecommerce tracking', function () {
       expect(window.dataLayer[1].search_results.term).toBe(onPageLoadExpected.search_results.term)
     })
 
+    it('should replace plusses with spaces, and remove extra lines/spaces from search query', function () {
+      onPageLoadExpected.search_results.term = 'i have a lot of spaces'
```

Comment:
Do you need to set this variable as part of the `onPageLoadExpected` object, or could it just be a `var` by itself?

Reply:
@andysellick Good spot, will change this in multiple tests :+1: